imagetest/fixtures.go

```diff
@@ -156,3 +158,39 @@ func (t *TestVM) EnableSecureBoot() {
 		}
 	}
 }
+
+// SetCustomNetwork set current test VMs in workflow using provided network and
+// subnetwork.
+func (t *TestVM) SetCustomNetwork(networkName, subnetworkName string) error {
+	for _, i := range t.testWorkflow.wf.Steps[createVMsStepName].CreateInstances.Instances {
+		if i.Name == t.name {
+			networkInterface := compute.NetworkInterface{
+				Network:    networkName,
+				Subnetwork: subnetworkName,
```

Comment:
what happens if this is omitted?

Reply:
the subnetwork field will not populated in daisy workflow

Reply:
I test this, it will just not populate subnetwork field. `"subnetwork": "test-subnet"`
> When `subnetwork = ""`, the instance will not populate subnetwork field.
> 
> ```
>           "StartupScript": "wrapper",
>             "disks": [
>               {
>                 "source": "vm2"
>               }
>             ],
>             "name": "vm2",
>             "networkInterfaces": [
>               {
>                 "accessConfigs": [
>                   {
>                     "type": "ONE_TO_ONE_NAT"
>                   }
>                 ],
>                 "aliasIpRanges": [
>                   {
>                     "ipCidrRange": "10.14.1.0/24",
>                     "subnetworkRangeName": "secondary-range"
>                   }
>                 ],
>                 "network": "test-net"
>               }
>             ],
> ```

Reply:
what will happen in the workflow if it is not populated?

Reply:
The workflow will just create vm with provided network with default subnetwork which has one subnetwork (subnet) in each region.

Reply:
in this case we created a network without automatic subnetting; won't omitting this value cause the workflow to fail in that case?
also with discussions, dont resolve til we've both finished with the discussion.

Reply:
no, the workflow is still valid in daisy and is still runnable.

Reply:
what subnet would it pick ? from docs: " If the network is in custom subnet mode, specifying the subnetwork is required."
https://cloud.google.com/compute/docs/reference/rest/v1/instances/insert

Reply:
oh. I see. you are talking about we set autoCreateSubnetworks to true, but not provide subnetwork. In this case, 
```
	<testsuite name="network-debian-10" tests="1" failures="0" errors="1" disabled="0" skipped="0" time="0">
		<system-err>step &#34;create-vms&#34; run error: googleapi: Error 400: Invalid value for field &#39;resource.networkInterfaces[0]&#39;: &#39;{  &#34;network&#34;: &#34;projects/hanga-testing/global/networks/test-net-network-9z878&#34;,  &#34;accessConfig&#34;: [{  ...&#39;. Subnetwork should be specified for custom subnetmode network, invalid</system-err>
	</testsuite>
```

Reply:
should we check for this case in this method?

Reply:
Add a check for this case and return error
```
2021/07/15 23:29:14 network.TestSetup for projects/debian-cloud/global/images/family/debian-10 failed: network test-net with custom mode must provide subnetwork
```